src/protocols/ethereum/libs/EthereumAdapter.ts

```diff
@@ -321,26 +324,75 @@ export class EthereumAdapter extends BaseProtocolAdapter {
     return normalized;
   }
 
-  override async fetchTransactions(
-    address: string,
-    nextPageParams?: string | null,
+  override async fetchAccountTransactions(
+    address: AccountAddress,
+    { nextPageNum }: ITransactionApiPaginationParams = {},
   ): Promise<IFetchTransactionResult> {
-    const { ethActiveNetworkPredefinedSettings } = useEthNetworkSettings();
-    const apiUrl = ethActiveNetworkPredefinedSettings.value.middlewareUrl;
-    const regularTransactions = await new EtherscanService(apiUrl)
-      .fetchAccountTransactions(address, nextPageParams || undefined);
+    const paginationParams: ITransactionApiPaginationParams = {};
+    let regularTransactions: ITransaction[] = [];
+
+    try {
+      const { ethActiveNetworkPredefinedSettings } = useEthNetworkSettings();
+      const service = new EtherscanService(ethActiveNetworkPredefinedSettings.value.middlewareUrl);
+      const responses = await Promise.all([
+        service.fetchAccountTransactions(address, { page: nextPageNum }),
+        service.fetchAccountTokenTransactions(address, { page: nextPageNum }),
+      ]);
+
+      regularTransactions = responses
+        .flat()
+        .filter(({ tx }) => +tx.amount > 0); // Remove transaction with 0 value (e.g.: fee payments)
```

Comment:
I thought we agreed to only remove transactions that are duplicate (only if another (actual) transaction with the same hash exists) and not filter them by whether they have 0 value.
@CedrikNikita  @peronczyk

Reply:
Yes, you are right. The hash suppose to be the same for two equal transactions that we got from different requests.

Reply:
The goal was to exclude the duplicates from `txlist` request by rewriting them with information from `tokentx` for the transactions with the same hash.

Reply:
Exactly, we should hide `fee payments transactions`. But filtering out all `0 value transactions` removes more than just the `fee payment transactions`.

Reply:
Fee payments are not duplicates - those are regular transactions with separate transaction hashes. Lets discuss this once again.

Reply:
For this list I propose to filter also out all fee payments transactions (what are the other 0 values?). We need to be able to show such information in origin transaction details page and/or aescan.

Reply:
I changed the logic so only when loading all account transactions the duplicated items with amount 0 are removed. Can you guys see if its OK now in your opinion?
@CedrikNikita @martinkaintas